temporian/core/operators/calendar_day.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Calendar operators."""
+
+from temporian.core import operator_lib
+from temporian.core.data import dtype
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class CalendarDayOperator(Operator):
+    """
+    Calendar operator to obtain the day of the month a timestamp belongs to.
+    """
+
+    def __init__(self, event: Event):
+        super().__init__()
+
+        # input
+        self.add_input("event", event)
+
+        output_feature = Feature(
+            name="calendar_day",
```

Comment:
I think calendar day its too generic. Could it be day of the week, day of the month, day of the year. Shouldn't be better to name it with something like `day_of_the_month` or `calendar_day_of_the_month`?

If we have **day of the week** with this naming convention it would be `calendar_week` and it's misleading if it's the day of the week (1-7) or the week of the year (1-52). In that case, I would prefer as well `day_of_the_week` or `calendar_day_of_the_week` instead of `calendar_week`.